docs/pipeline-components-and-applications/loaders-storage-targets/lake-loader/_overview.md

```diff
@@ -0,0 +1,33 @@
+```mdx-code-block
+import Mermaid from '@theme/Mermaid';
+import {versions} from '@site/src/componentVersions';
+import CodeBlock from '@theme/CodeBlock';
+```
+
+<p>The lake loader on {props.cloud} is a fully streaming application that continually pulls events from {props.stream} and writes to {props.bucket}.</p>
+
+<Mermaid value={`
+flowchart LR
+  stream[["<b>Enriched Events</b>\n(${props.stream} stream)"]]
+  loader{{"<b>Lake Loader</b>"}}
+  subgraph bucket ["${props.bucket}"]
+    table[("<b>Events table</b>")]
+  end
+  stream-->loader-->bucket
+`}/>
+
+<p>The lake loader is published as a Docker image which you can run on any {props.cloud} VM.  You do not need a Spark cluster to run this loader.</p>
```

Comment:
This should not be in the same reusable block as the chart. The reason is that we then need to import the chart here: https://docs.snowplow.io/docs/storing-querying/loading-process/